Libs/MRML/Core/vtkMRMLSliceNode.h

```diff
@@ -23,6 +23,15 @@ class vtkMRMLVolumeNode;
 class vtkMatrix3x3;
 class vtkMatrix4x4;
 
+/// \brief Structure to store preset orientation information.
+struct OrientationPresetType
+{
+  std::string Name;
+  vtkSmartPointer<vtkMatrix3x3> Orientation;
+  std::string Prefix;
+  std::string Tooltip;
```

Comment:
@lassoan  What about changing `Tooltip` into `Description` ?

Reply:
Agreed. It makes sense to update `qMRMLViewControllerBar` to generate reasonable default for Prefix, Tooltip, ...

That said, to support custom value per presets:
* for a custom preset named `Foo`, add  user setting entry (e.g `ViewControllerBar/OrientationPresets/Foo/Tooltip`, `ViewControllerBar/OrientationPresets/Foo/Prefix`)
* the qMRMLViewControllerBar would be updated to check if settings are set and associated values 

Finally, if in a custom application more advanced customization are still needed, making sure the  "widgets" of the viewer controller are associated with an objectName allow getting their reference and update them. (for example to hide the pin button, set prefix to empty string, ...)

Reply:
> to support custom value per presets

Allowing setting custom positive/negative axis labels should be enough customization. Do you have any use case where this would not give good results?

> Finally, if in a custom application more advanced customization are still needed, making sure the "widgets" of the viewer controller are associated with an objectName allow getting their reference and update them.

I agree. This would allow fine-grained customization without overcomplicating the API. For common customizations, such as hiding the pin button and adding custom buttons next to the slider, we should also add explicit method calls.